groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group7/ManganeseChain.groovy

```diff
@@ -31,15 +33,15 @@ for (combustible in combustibles()) {
 
 ROASTER.recipeBuilder()
```

Comment:
rhodochrosite should produce chemically pure MnO simply by regulating the partial pressure of oxygen in the atmosphere.

Reply:
this MnO produced should be suitable for most chemical applications, instead clarify a "battery grade" MnO for the high purities required for battery applications.

Reply:
done